hello.html

```diff
@@ -0,0 +1,56 @@
+<html>
+<a href="http://hellojalenmerch.com/">Get your merchandise here! </a>
+<head>
+  <title>Welcome to Hello Jalen</title>
+  <style>
+    h1{
+      color:red;
+    }
+    p{
+      border: 10px dotted red;
+      font-size: 32px;
+    }
+  </style>
+  </head>
+  <body>
+<head>
```

Comment:
See https://learn.shayhowe.com/html-css/building-your-first-web-page/ for the basic structure of an HTML document. It only contains one `<head>` element and one `<body>` element. The `<body>` element is never inside the `<head>` element.